### ISSUE TITLE
WebCal events not displayed in Grid

### ISSUE THREAD

**Issue by marbetschar** [elementary/calendar#665]
## What Happened

I added a calendar subscription to my Nextcloud backend in order to display holidays in elementary Calendar. To achieve this, I copied the iCal URL from here https://www.feiertagskalender.ch/export.php?geo=1638&jahr=2021&klasse=5&hl=de and added it to my Nextcloud.

Then I resynced in Evolution and the calendar and its events appeared there correctly:

![Screenshot from 2021-05-10 15-38-27](https://user-images.githubusercontent.com/392542/117668802-93f61e80-b1a6-11eb-9539-ee8f7ec3750e.png)

However, elementary Calendar does seem to render the events in the sidebar - but does not show them in the main event grid:

![Screenshot from 2021-05-10 15-32-07](https://user-images.githubusercontent.com/392542/117668923-ae2ffc80-b1a6-11eb-8ebf-018063c177d6.png)

## Platform Information

I'm running the elementary OS 6 Pre-Release version.

**Comment by mcclurgm** [comment]
Yeah, that's… weird. I have a couple questions so I understand what's going on. 

- To be clear, you added this calendar to Nextcloud, then how did you get the Nextcloud calendar into the elementary Calendar app? Is there something like a per-account sync in Evolution that automatically detected the new calendar, or did you add it manually?
- Do you see this behavior with other Nextcloud calendars?
- Is there anything in the debug log? (In case there's a null error or something like that.) 

Since I don't use Nextcloud, I'm not quite sure how to debug this. Are you aware of any way for me to get a similar calendar, or are there any public Nextcloud calendars I could try out (if such a thing exists) that you've seen display this issue?
